libs/remix-debug/test/decoder/vmCall.ts

```diff
@@ -16,14 +17,14 @@ export function sendTx (vm, from, to, value, data, cb) {
     data: Buffer.from(data, 'hex')
   })
   tx.sign(from.privateKey)
-  var block = new Block({
-    header: {
-      timestamp: new Date().getTime() / 1000 | 0,
-      number: 0
-    },
-    transactions: [],
-    uncleHeaders: []
+
+  var header = BlockHeader.fromHeaderData({
+    timestamp: new Date().getTime() / 1000 | 0,
+    number: 0
   })
+
+  var block = new Block(header, [], [])
```

Comment:
to simplify here you can just do:

```suggestion
  var block = Block.fromBlockData({
    header: {
      timestamp: new Date().getTime() / 1000 | 0,
      number: 0
    }
  })
```